lib/modules/ssh/strong_password/__init__.py

```diff
@@ -1,9 +1,26 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
+import time
 from core.compatible import generate_token
 
 
+class ModuleProcessor:
+    """
+    this is the processor to run after docker machine is up to grab the log files or do other needed process...
+    """
+    def __init__(self):
+        self.kill_flag = False
+
+    def processor(self):
+        """
+        processor function will be called as a new thread and will be die when kill_flag is True
+        :return:
+        """
+        while not self.kill_flag:
```

Comment:
submit the info from logins.txt to mongodb

Reply:
as we are sharing the file with -v ,should i write code to connect to mongodb in container or from docker host?

Reply:
from docker host of course, in the same loop!

Reply:
that's why this class/function exist.

Reply:
oh yes. then i would need to access db config here and connect it from here and insert into honeypot events right?